### PR TITLE
Add Teleport

### PR DESCRIPTION
**Pull request by benarent**
Different branch naming

### CODE REVIEW

**Comment by phallobst** [comment]
Renaming the branch didn't work either. :frowning_face:

**Comment by benarent** [comment]
super strange, I also tried an older github pages gem but no luck

**Comment by Carlgo11** [comment]
Tried to test a branch the same as master and one before the latest commit but both [failed](https://github.com/Carlgo11/twofactorauth/runs/7814117606?check_suite_focus=true). Seems to not be related to this PR

**Comment by Carlgo11** [comment]
Managed to get one build to [succeed](https://github.com/Carlgo11/twofactorauth/runs/7814265104?check_suite_focus=true#step:6:20). @phallobst I got a PR for you to look at in a few secs :wink:

**Comment by phallobst** [comment]
@benarent I think you should be able to rebase your branch on upstream/master now without any checks failing.

File: img/g/goteleport.com.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
+        </g>
+    </g>
+</svg>
```
Comment (Carlgo11) [inline img/g/goteleport.com.svg]: Here's a minified version of the SVG
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 96 98"><path fill="#512FC9" d="M38.4 1a49.8 49.8 0 0 1 19.2 0 2 2 0 0 1 1.7 1.5l2.4 12.3a38 38 0 0 1 10.1 5.7L84 16.3a2 2 0 0 1 2.2.6A46.6 46.6 0 0 1 95.8 33c.4.8.1 1.7-.5 2.2l-9.7 8.1a35.6 35.6 0 0 1 0 11.4l9.7 8.1c.6.5.9 1.4.5 2.2-2 5.9-5.3 11.5-9.6 16.1-.5.7-1.4 1-2.2.6l-12.2-4.1c-3 2.4-6.4 4.3-10 5.7l-2.5 12.2a2 2 0 0 1-1.7 1.6 53.8 53.8 0 0 1-19.2 0 2 2 0 0 1-1.7-1.6l-2.4-12.2a36.2 36.2 0 0 1-10.1-5.7l-12.2 4c-.8.4-1.6.2-2.2-.5A46.4 46.4 0 0 1 .2 65c-.4-.8-.1-1.7.6-2.2l9.6-8a38.6 38.6 0 0 1 0-11.5L.8 35.2c-.7-.5-1-1.4-.6-2.2 2-5.8 5.3-11.4 9.6-16 .6-.8 1.4-1 2.2-.7l12.2 4.2c3-2.4 6.4-4.4 10.1-5.7l2.4-12.3A2 2 0 0 1 38.4 1ZM48 20.2A29 29 0 0 0 18.8 49 29 29 0 0 0 48 77.7 29 29 0 0 0 77.2 49 29 29 0 0 0 48 20.3Zm18.5 16.2V45H52.4v20.5h-8.8V45h-14v-8.5h36.9Z"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/g/goteleport.com.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "Teleport": {
+      "domain": "goteleport.com",
+      "tfa": [
+        "totp",
+        "u2f"
+      ],
+      "documentation": "https://goteleport.com/docs/access-controls/guides/webauthn/",
+      "recovery": "https://github.com/gravitational/teleport/discussions/12379",
```
Comment (phallobst) [inline entries/g/goteleport.com.json]: The `recovery` entry should include the documentation for users that cannot log in anymore - but these are instructions to generate recovery codes beforehand.
```suggestion
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Comment by benarent** [comment]
Thanks @phallobst & @Carlgo11 . I've made the changes and we're green now!

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**